### PR TITLE
COMMANDS.md: reflect modlog changes

### PR DESCRIPTION
**Pull request by sparkychildcharlie**

### CODE REVIEW

**Comment by Zarel** [comment]
I would not leave "removed" messages. Just document the current API.

File: chat-plugins/COMMANDS.md [hunk 1]
```diff
@@ -110,12 +110,12 @@ Commands have access to the following functions:
...
-`this.addModCommand(message)`
-*	Like this.add, but also logs the message to the moderator log which can
-	be seen with `/modlog`.
+`this.addModAction(message)`
+*	Like this.add, except it logs the message as being sent as the user who used the command.
+	This does not log anything into the modlog
 
-`this.logModCommand(message)`
-*	Like `this.addModCommand`, except users in the room won't see it.
+`modlog(action, user, note, options)`
```
Comment (Zarel) [inline chat-plugins/COMMANDS.md]: ahem

`this.modlog`

Comment (Zarel) [reply chat-plugins/COMMANDS.md]: Also point out that `note` and `options` and `user` are optional.

Comment (sparkychildcharlie) [reply chat-plugins/COMMANDS.md]: right thanks for catching that - i'm dumb

**Comment by Zarel** [comment]
You should probably actually document `options`, come to think of it.

**Comment by Zarel** [comment]
Unmaintained, but I guess still an improvement over the current state. I still want better documentation for `options`.

**Comment by sparkychildcharlie** [comment]
Oh, my bad - sorry; I was too busy with homework and assignments and forgot
about this one

**Pull request merged**